### ISSUE TITLE
Windows build issues

### ISSUE THREAD

**Issue by brianjo** [pytorch/tutorials#1383]
Hi @guyang3532, 

Could you look at the windows build and see if we can fix the errors that are occuring? 

The build seems to be working fine on Linux. Thanks! 

https://app.circleci.com/pipelines/github/pytorch/tutorials?branch=master

@malfet

**Comment by brianjo** [comment]
There's an issue with the librosa library. I think you may need to include the binaries using your script. See the windows note here. https://pypi.org/project/librosa/

**Comment by skyline75489** [comment]
Hi @brianjo. After discussion with @guyang3532 , we found that it is very difficult to make `audio_preprocessing_tutorial.py` work in the CI environment due to its usage of librosa (gstreamer) and multiprocessing module. So I filed #1414 to exclude it from the Windows CI. Is this a feasible solution to you?  

<hr>

To be specific, first, the multiprocessing module behaves differently on Windows, causing the `process.start()` to fail. See [example run](https://app.circleci.com/pipelines/github/pytorch/tutorials/3638/workflows/76e5bb47-d383-476f-9e85-cb0934365d6e/jobs/68223). This seems to only happen during sphinx build. 

Second, the gstreamer binaries you mentioned is not used by conda. I tried this locally by installing all the dependencies and do `python audio_preprocessing_tutorial.py`. The MP3 file is still not recognized. Even if I install the gstreamer using anaconda (`conda install -c conda-forge gstreamer`), the code still fails. I think it might be because there's some other requirements on Windows for librosa to work.
